django/contrib/admindocs/views.py

```diff
@@ -411,8 +412,7 @@ def simplify_regex(pattern):
     """
     pattern = replace_named_groups(pattern)
     pattern = replace_unnamed_groups(pattern)
-    # clean up any outstanding regex-y characters.
```

Comment:
I think we should leave an updated comment: `# Remove any outstanding regex metacharacters.`